haystack/components/generators/utils.py

```diff
@@ -11,4 +13,12 @@ def print_streaming_chunk(chunk: StreamingChunk) -> None:
 
     Prints the tokens of the first completion to stdout as soon as they are received
```

Comment:
Let's update the docstrings to reflect that this supports streaming tool calls in addition to normal text